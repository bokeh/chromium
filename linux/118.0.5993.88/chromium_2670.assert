type: account-key
authority-id: canonical
revision: 2
public-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul
account-id: canonical
name: store
since: 2016-04-01T00:00:00.0Z
body-length: 717
sign-key-sha3-384: -CvQKAwRQ5h3Ffn10FILJoEZUXOv6km9FwA80-Rcj-f-6jadQ89VRswHNiEB9Lxk

AcbBTQRWhcGAARAA0KKYYQWuHOrsFVi4p4l7ZzSvX7kLgJFFeFgOkzdWKBTHEnsMKjl5mefFe9ji
qe8NlmJdfY7BenP7XeBtwKp700H/t9lLrZbpTNAPHXYxEWFJp5bPqIcJYBZ+29oLVLN1Tc5X482R
vCiDqL8+pPYqBrK2fNlyPlNNSum9wI70rDDL4r6FVvr+osTnGejibdV8JphWX+lrSQDnRSdM8KJi
UM43vTgLGTi9W54oRhsA2OFexRfRksTrnqGoonCjqX5wO3OFSaMDzMsO2MJ/hPfLgDqw53qjzuKL
Iec9OL3k5basvu2cj5u9tKwVFDsCKK2GbKUsWWpx2KTpOifmhmiAbzkTHbH9KaoMS7p0kJwhTQGA
o9aJ9VMTWHJc/NCBx7eu451u6d46sBPCXS/OMUh2766fQmoRtO1OwCTxsRKG2kkjbMn54UdFULl9
VfzvyghMNRKIezsEkmM8wueTqGUGZWa6CEZqZKwhe/PROxOPYzqtDH18XZknbU1n5lNb7vNfem9F
2ai+3+JyFnW9UhfvpVF7gzAgdyCqNli4C6BIN43uwoS8HkykocZS/+Gv52aUQ/NZ8BKOHLw+7ant
Q0o8W9ltSLZbEMxFIPSN0stiZlkXAp6DLyvh1Y4wXSynDjUondTpej2fSvSlCz/W5v5V7qA4nIcG
vUvV7RjVzv17ut0AEQEAAQ==

AcLDXAQAAQoABgUCV83k9QAKCRDUpVvql9g3IBT8IACKZ7XpiBZ3W4lqbPssY6On81WmxQLtvsMV
WTp6zZpl/wWOSt2vMNUk9pvcmrNq1jG9CuhDfWFLGXEjcrrmVkN3YuCOajMSPFCGrxsIBLSRt/bP
nrKykdLAAzMfG8rP1d82bjFFiIieE+urQ0Kcv09Jtdvavq3JT1Tek5mFyyfhHNlQEKOzWqmRWiLg
3c3VOZUs1ZD8TSlnuq/x+5T0X0YtOyGjSlVxk7UybbyMNd6MZfNaMpIG4x+mxD3KHFtBAC7O6kLe
eX3i6j5nCY5UABfA3DZEAkWP4zlmdBEOvZ9t293NaDdOpzsUHRkoi0Zez/9BHQ/kwx/uNc2WqrYm
inCmu16JGNeXqsyinnLl7Ghn2RwhvDMlLxF6RTx8xdx1yk6p3PBTwhZMUvuZGjUtN/AG8BmVJQ19
rsGSRkkSywvnhVJRB2sudnrMBmNS2goJbzSbmJnOlBrd2WsV0T9SgNMWZBiov3LvU4o2SmAb6b+k
rYwh8H5QHcuuYJuxDjFhPswIp6Wes5T6hUicf3SWtObcDS4HSkVS4ImBjjX9YgCuFy7QdnooOWEY
aPvkRw3XCVeYq0K6w9GRsk1YFErD4XmXXZjDYY650MX9v42Sz5MmphHV8jdIY5ssbadwFSe2rCQI
6UX08zy7RsIb19hTndE6ncvSNDChUR9eEnCm73eYaWTWTnq1cxdVP/s52r8uss++OYOkPWqh5nOu
haRn7INjH/yZX4qXjNXlTjo0PnHH0q08vNKDwLhxS+D9du+70FeacXFyLIbcWllSbJ7DmbumGpFo
yYbtj3FDDPzachFQdIG3lSt+cSUGeyfSs6wVtc3cIPka/2Urx7RprfmoWSI6+a5NcLdj0u2z8O96
HxeIgxDpg/3gT8ZIuFKePMcLDM19Fh/p0ysCsX+84B9chNWtsMSmIaE57V+959MVtsLu7SLb9gi7
skrju0pQCwsu2wHMLTNd1f3PTHmrr49hxetTus07HSQUApMtAGKzQilF5zqFjbyaTd4xgQbd+PKW
CjFyzQTDOcUhXpuUGt/IzlqiFfsCsmbj2K4KdSNYMlqIgZ3Azu8KvZLIhsyN7v5vNIZSPfEbjdeu
ClU9r0VRiJmtYBUjcSghD9LWn+yRLwOxhfQVjm0cBwIt5R/yPF/qC76yIVuWUtM5Y2/zJR1J8OFq
qWchvlImHtvDzS9FQeLyzJAOjvZ2CnWp2gILgUz0WQdOk1Dq8ax7KS9BQ42zxw9EZAEPw3PEFqRy
IQsRTONp+iVS8YxSmoYZjDlCgRMWUmawez/Fv5b9Fb/XkO5Eq4e+KfrpUujXItaipb+tV8h5v3tr
oG3Ie3WOHrVjCLXIdYslpL1O4nadqR6Xv58pHj6k

type: snap-declaration
format: 4
authority-id: canonical
revision: 19
series: 16
snap-id: XKEcBqPM06H1Z7zGOdG5fbICuf8NWK5R
plugs:
  audio-record:
    allow-auto-connection: true
  bluez:
    allow-auto-connection: true
  browser-support:
    allow-auto-connection: true
  camera:
    allow-auto-connection: true
  cups-control:
    allow-auto-connection: true
  joystick:
    allow-auto-connection: true
  personal-files:
    allow-auto-connection: true
    allow-installation:
      -
        plug-attributes:
          read: \$HOME/\.config/chromium
        plug-names:
          - chromium-config
      -
        plug-attributes:
          read: \$HOME/\.(config/chromium|chromium-browser\.init)
        plug-names:
          - chromium-config
      -
        plug-attributes:
          write: \$HOME/\.local/share/applications
        plug-names:
          - dot-local-share-applications
      -
        plug-attributes:
          write: \$HOME/\.local/share/icons
        plug-names:
          - dot-local-share-icons
  removable-media:
    allow-auto-connection: true
  system-files:
    allow-auto-connection: true
    allow-installation:
      plug-attributes:
        read: /etc/chromium-browser/policies
      plug-names:
        - etc-chromium-browser-policies
  system-packages-doc:
    allow-auto-connection: true
  u2f-devices:
    allow-auto-connection: true
publisher-id: canonical
snap-name: chromium
timestamp: 2023-09-26T13:11:47.201611Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCZRLYkwAAgzMQABkslB/nsmTQC5f2RN8Q6hzOjXj0szIflPJCwxMTZoyfjpTI
ftgEMrB2SlQ3ZAY5VwIXg/bTyUqebfK4G/d+sRhwbeGBr5PnpJ1K/7ON9oEw4LMYWIUbovEJjkuh
YLghtr3G1iyzhiF6+nJ5QuTkPppCa6eL28K5ZxDn3BRAtYoHRf29IbKr8HzBfTmDhgyz0083efIe
MX4J5fIM1KfIe18e/3e9xlm+MoLdMSO4hkrqWQjZQPtql3932goExaG2Gm1GvEJb7CXr0dahNbuF
PEVSN4vYIFOsojmOv4ptBvBSSli7iDymEZ/f0YXF+/k61njAEYVdQQt6GnsnY9Vb23/p2MAQQoUe
bNi3CPRReOjGTrsrY78Q3NG61I99376P/zi56XN4/QLnhaadnqi/qsyi6KfUkJpPHk7YI/ukj/J1
oGT55CQJjim1+VpPc0JujVn+Zak+Sz4/DprWcDVLvK4JMXxrxG+OgHWaCPM1Y89M5nnnPc+/KfYO
6lmbrQJdTh6V3qYBjqpi2TuXLSMIydExm4QMeY+DORZPEbxzdShr2N6U+QxpDyZRsrFaAqAD+a1P
zmQGZFb8hL6+zB1DlC3B/Mm388iKKDPySbfeo6luk4fDTNyUdSDthRSe2m0MYikkHyi6I2qVcBQh
MLU9AHClbRNtDBY+ZlrJnLg9B5xM

type: snap-revision
authority-id: canonical
snap-sha3-384: NgwSEtLcSJnRmFg1V5iCY0YZAGy9l0NwKdaHq4CKd5TLYPhMBcj2zC2wglmNgRWi
developer-id: canonical
provenance: global-upload
snap-id: XKEcBqPM06H1Z7zGOdG5fbICuf8NWK5R
snap-revision: 2670
snap-size: 163442688
timestamp: 2023-10-18T17:24:51.466567Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCZTAU4wAALlUQAM0018rlmZLTRyyHodBV6mY2qJ4kLSjbbAJzG0DtYbPB/jPC
G29PsquMSKe72nGLCMrTm6iYcuO1mr5ZmKTvY4SIYb3AXbS1Fz+6bFnFYeljG/ZTgGlqLkrQt9Ng
buVw1LIzfe0OJM6RwHIbMzqreDzdjJDgbIIz/vpG/cPDDXhxaRiaTRqopJ3ILwRR967xndQP/4k6
PNQWZ1uTIHeA52VSUoEDL3KsuhfB4LjNrPSNdAfhRbXxhxiB2Rqvlb9f5kFYSLVAPZdHYRPQVP3P
awvFIldqxmWwZzY1/6YM1tBU10M7jQ+O/amEJdfEltCEUH0P1EE5YtX/NluuAX2eKTZjn1qvAIkP
6hshxjyoVeA51lKBa9vvf8KvLo/R6zT8HPp/DWiFRMa+Gro2B7HFkaL3cPDEo/HZzlakj1cEnOVN
ejrAJkQYsU9MAEWXmyNUHZnkjknkiaDuJ45pTTnVjYrpv9kAHumiE5xgRlG0YuVB0D1xBo/6yy5A
Mxkx+KktCZkeSQnPuIyp0NILbBp4wxhFY1rfZ7lYokFpqqm4u2TLyhaqrGeYLeKEGEF9RM3OPnsn
a6c03iPOkfsWgI8Ut3xTYUl+U03LUGJpdbCNjDO2zUzAVeC2xidNuaO4hFuuQf+4vkfcOF6uGuUx
18Pri49h+5m53nmzYHkyb006SQsW
