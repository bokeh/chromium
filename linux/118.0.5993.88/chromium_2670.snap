version https://git-lfs.github.com/spec/v1
oid sha256:09cdb505086d875631709eea08c60aa79001124d68bfb2f14d15fb3f3c2b85cd
size 163442688
